install/kubernetes/cilium/values.yaml

```diff
@@ -325,6 +325,9 @@ global:
     # ports is detected
     autoProtectPortRange: true
 
+    # enableHealthCheckNodeport enables healthcheck nodePort server for NodePort services
+    enableHealthCheck: "true"
```

Comment:
Nit: we don't use quotes in values.yaml.

Reply:
I think the 'helm lint' test was not passing without the quotes. I will recheck

Reply:
Another nit: The comment still refers to it as enableHealthCheckNodeport